docs/sources/tempo/metrics-from-traces/metrics-queries/configure-traceql-metrics.md

```diff
@@ -88,6 +88,15 @@ Refer to the [Helm chart for an example](https://github.com/grafana/helm-charts/
 
 For more information about overrides, refer to [Standard overrides](https://grafana.com/docs/tempo/<TEMPO_VERSION>/configuration/#standard-overrides).
 
+### Local blocks and metrics-generator in Azure blob storage
```

Comment:
since the file for this is irrelevant, we can remove this

Reply:
This is specifically for Helm chart stuff. I'll make sure it'c early labeled.